scripts/transactions_exporter.py

```diff
@@ -1,63 +1,73 @@
+import logging
 import time
 import warnings
 from decimal import Decimal
-import logging
 
 import pandas as pd
 from brownie import ZERO_ADDRESS, Contract, chain, web3
 from brownie.exceptions import BrownieEnvironmentWarning
+from pony.orm import db_session
 from joblib import Parallel, delayed
-import sqlalchemy
-from tqdm import tqdm
 from web3._utils.abi import filter_by_name
 from web3._utils.events import construct_event_topic_set
-from yearn.events import create_filter, decode_logs, get_logs_asap
-from yearn.outputs.postgres.postgres import postgres
+from yearn.entities import UserTx  # , TreasuryTx
+from yearn.events import decode_logs, get_logs_asap
+from yearn.outputs.postgres.utils import (cache_address, cache_token,
+                                          last_recorded_block)
 from yearn.prices import magic
-from yearn.utils import contract
-from yearn.v1.registry import Registry as RegistryV1
-from yearn.v2.registry import Registry as RegistryV2
-from yearn.treasury.treasury import Treasury
-
-treasury = Treasury()
+from yearn.yearn import Yearn
 
 warnings.simplefilter("ignore", BrownieEnvironmentWarning)
 
-registryV1 = RegistryV1()
-registryV2 = RegistryV2()
+yearn = Yearn(load_strategies=False)
 
 logger = logging.getLogger(__name__)
 
 
 def main():
     for block in chain.new_blocks(height_buffer=1):
-        process_and_cache_user_txs(postgres.last_recorded_block('user_txs'))
-
-
-def active_vaults_at(end_block) -> set:
-    v1 = {vault.vault for vault in registryV1.active_vaults_at(end_block)}
-    v2 = {vault.vault for vault in registryV2.active_vaults_at(end_block)}
-    return v1.union(v2)
+        process_and_cache_user_txs(last_recorded_block(UserTx))
 
 
+@db_session
 def process_and_cache_user_txs(last_saved_block=None):
-    max_block_to_cache = (
-        chain.height - 50
-    )  # We look 50 blocks back to avoid uncles and reorgs
+    # NOTE: We look 50 blocks back to avoid uncles and reorgs
+    max_block_to_cache = chain.height - 50
     start_block = last_saved_block + 1 if last_saved_block else None
     end_block = (
-        10650000
+        9480000
```

Comment:
what's that magic number ? maybe add a comment or extract as constant

Reply:
its just an arbitrary number that slightly lags behind the first iearn deployments, didn't feel right to write that in a comment but I can do that

Reply:
done